packages/openapi-v3/src/decorators/parameter.decorator.ts

```diff
@@ -437,6 +443,70 @@ export namespace param {
       schema: {type: 'array', items: itemSpec},
     });
   };
+
+  /**
+   * Sugar decorator for `filter` query parameter
+   *
+   * @example
+   * ```ts
+   * async find(
+   *   @param.filter(modelCtor)) filter?: Filter<T>,
+   * ): Promise<(T & Relations)[]> {
+   *   // ...
+   * }
+   * ```
+   * @param modelCtor - Model class
+   * @param name - Custom name for the parameter, default to `filter`
+   *
+   */
+  export function filter(modelCtor: typeof Model, name = 'filter') {
+    return param.query.object(name, getFilterSchemaFor(modelCtor));
+  }
+
+  /**
+   * Sugar decorator for `filter` query parameter excluding `where`
+   *
+   * @example
+   * ```ts
+   * async findById(
+   *   @param(idPathParam) id: IdType,
+   *   @param.filter(modelCtor)) filter?: FilterExcludingWhere<T>,
+   * ): Promise<(T & Relations)[]> {
+   *   // ...
+   * }
+   * ```
+   * @param modelCtor - Model class
+   * @param name - Custom name for the parameter, default to `filter`
+   *
+   */
+  export function filterExcludingWhere(
+    modelCtor: typeof Model,
+    name = 'filter',
+  ) {
```

Comment:
Similarly here, have you considered adding `options` argument to `@param.filter`, instead of introducing another decorator?